# Translations template for cinder.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the cinder project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Cinder\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-11-11 06:09+0000\n"
"PO-Revision-Date: 2014-11-10 13:52+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Korean (Korea) (http://www.transifex.com/projects/p/cinder/"
"language/ko_KR/)\n"
"Language: ko_KR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: cinder/backup/manager.py:445
msgid "Failed to update usages deleting backup"
msgstr ""

#: cinder/backup/manager.py:629
msgid "Backup driver has not been initialized"
msgstr ""

#: cinder/backup/drivers/ceph.py:459
#, python-format
msgid ""
"Max retries reached deleting backup %(basename)s image of volume %(volume)s."
msgstr ""

#: cinder/backup/drivers/ceph.py:495
#, python-format
msgid "Pipe1 failed - %s "
msgstr ""

#: cinder/backup/drivers/ceph.py:509
#, python-format
msgid "Pipe2 failed - %s "
msgstr ""

#: cinder/backup/drivers/ceph.py:973
msgid "Differential restore failed, trying full restore"
msgstr ""

#: cinder/backup/drivers/ceph.py:1175
#, python-format
msgid "Restore to volume %(volume)s finished with error - %(error)s."
msgstr ""

#: cinder/backup/drivers/swift.py:159
#, python-format
msgid "single_user auth mode enabled, but %(param)s not set"
msgstr ""

#: cinder/backup/drivers/swift.py:394
#, python-format
msgid "Backup volume metadata to swift failed: %s"
msgstr ""

#: cinder/backup/drivers/tsm.py:252
#, python-format
msgid ""
"backup: %(vol_id)s failed to remove backup hardlink from %(vpath)s to "
"%(bpath)s.\n"
"stdout: %(out)s\n"
" stderr: %(err)s."
msgstr ""

#: cinder/backup/drivers/tsm.py:531
#, python-format
msgid ""
"delete: %(vol_id)s failed with stdout: %(out)s\n"
" stderr: %(err)s"
msgstr ""

#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:50
#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:75
#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:106
#, python-format
msgid "Table |%s| not created!"
msgstr "|%s| 테이블이 작성되지 않았습니다!"

#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:128
msgid "Dropping foreign key reservations_ibfk_1 failed."
msgstr ""

#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:134
msgid "quota_classes table not dropped"
msgstr "quota_classes 테이블이 삭제되지 않았음"

#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:141
msgid "quota_usages table not dropped"
msgstr "quota_usages 테이블이 삭제되지 않았음"

#: cinder/db/sqlalchemy/migrate_repo/versions/002_quota_class.py:148
msgid "reservations table not dropped"
msgstr "예약 테이블이 삭제되지 않았음"

#: cinder/db/sqlalchemy/migrate_repo/versions/015_drop_migrations_table.py:32
msgid "migrations table not dropped"
msgstr ""

#: cinder/db/sqlalchemy/migrate_repo/versions/015_drop_migrations_table.py:62
#, python-format
msgid "Table |%s| not created"
msgstr ""

#: cinder/db/sqlalchemy/migrate_repo/versions/021_add_default_quota_class.py:76
msgid "Default quota class data not inserted into the DB."
msgstr ""

#: cinder/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "기존 예외가 삭제됨: %s"

#: cinder/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "예기치 않은 예외 %d 번 발생하였습니다... 다시 시도중."

#: cinder/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "고정 기간 루프 호출에서"

#: cinder/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "동적 루프 호출에서"

#: cinder/openstack/common/periodic_task.py:202
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr "%(full_task_name)s 중 오류: %(e)s"

#: cinder/openstack/common/policy.py:508
#, python-format
msgid "Failed to understand rule %s"
msgstr "%s 규칙을 이해하는데 실패했습니다"

#: cinder/openstack/common/policy.py:518
#, python-format
msgid "No handler for matches of kind %s"
msgstr "%s 유형의 일치에 대한 핸들러가 없음"

#: cinder/openstack/common/policy.py:788
#, python-format
msgid "Failed to understand rule %r"
msgstr "룰 %r를 이해하지 못했습니다."

#: cinder/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr "RPC 정리하는 동안 예외입니다."

#: cinder/openstack/common/service.py:277
msgid "Unhandled exception"
msgstr "처리되지 않은 예외"
